modules/oe_content_entity/tests/src/FunctionalJavascript/DocumentReferenceCorporateEntityUiTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_entity\FunctionalJavascript;
+
+/**
+ * Test Document reference content entity UIs.
+ */
+class DocumentReferenceCorporateEntityUiTest extends CorporateEntityUiTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'oe_content',
+    'oe_content_entity',
```

Comment:
not necessary. should be handled by dependencies of module.

Reply:
indeed.